setup.py

```diff
@@ -2,19 +2,42 @@
 
 from setuptools import setup
 
+import i18n
```

Comment:
@nedbat The import had to be moved into __call__ because in just trying to import i18n, it was importing all of config and then failing on not being able to import yaml. I based the importing i18n into the setup file from the edx-drf-extensions repo.